graphene_sqlalchemy/utils.py

```diff
@@ -26,7 +26,12 @@ def get_query(model, context):
 def is_mapped_class(cls):
     try:
         class_mapper(cls)
-    except (ArgumentError, UnmappedClassError):
+    except ArgumentError as error:
+        # Only handle ArgumentErrors for non-class objects
+        if "Class object expected" in str(error):
```

Comment:
To me, this is a bit of a code-smell. Is it possible to instead create a new exception class and catch that instead? I'm not sure what the import tree looks like, so I'm unsure where it should go, but perhaps:
```
class InvalidSQLAlchemyModel(ValueError):
    pass

...

    raise InvalidSQLAlchemyModel("You need to pass a valid SQLAlchemy Model in ... ")
```
Then there's no need for checking the string of the error message here.

Reply:
I agree but I think that would need to be handled in SQLAlchemy itself